packages/web/src/components/spa/Navigation/MobileNav/NavMobile.jsx

```diff
@@ -122,9 +105,8 @@ export const NavMobile = ({ props }) => {
                 className={menuClass}
                 style={{
                     height: bodyHeight,
-                    backgroundColor: props.mobileBgColor
-                }}
-            >
+                    backgroundColor: props.mobileBgColor,
```

Comment:
Why are there so many linting changes?

Reply:
Do you have the precommit hooks installed or you're using some custom linter? Make sure you run `npm run prepare` in the root directory of the project.

Reply:
No idea, everything is up to date and installed.
![Screenshot 2024-02-21 at 17 43 25](https://github.com/AUBGTheHUB/monolith/assets/83674495/b054ce20-28a0-4593-a802-42e04be18780)